README.md

```diff
@@ -1,19 +1,91 @@
 # CodestoriesReviewBot
 
-To start your Phoenix server:
+## :notebook: Description
 
-  * Install dependencies with `mix deps.get`
-  * Create and migrate your database with `mix ecto.setup`
-  * Start Phoenix endpoint with `mix phx.server`
+Our awesome bot to trade Codestories drafts! 🚀
 
-Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
+## :closed_lock_with_key: Technology
 
-Ready to run in production? Please [check our deployment guides](https://hexdocs.pm/phoenix/deployment.html).
+| Name |  Version |
+| :--: | :---: |
+| [Elixir](https://elixir-lang.org/) | 1.7.4 |
+| [Phoenix](https://phoenixframework.org/) | 1.4.0 |
+| [PostgreSQL](http://www.postgresql.org/) | 10.6 |
 
-## Learn more
+## :hammer: Setup
 
-  * Official website: http://www.phoenixframework.org/
-  * Guides: https://hexdocs.pm/phoenix/overview.html
-  * Docs: https://hexdocs.pm/phoenix
-  * Mailing list: http://groups.google.com/group/phoenix-talk
-  * Source: https://github.com/phoenixframework/phoenix
+### :information_source: Installing dependencies
+
+* Install Elixir for MacOS X
+
+```
+brew update
+brew install elixir
+elixir -v
+brew upgrade elixir # use if if the previous steps returns version below 1.4
+```
+
+* Install Hex - Dependencies Manager
+
+```
+mix local.hex
+```
+
+* Install all dependencies used by the Project
+
+```
+$ cd codestories-review-bot
+$ mix deps.get
+```
+
+### :pencil: Default configuration
+
+Define local values for the environment variables defined in `config\dev.exs` file.
+
+### :rocket: Running server on the local machine
+
+Then configure your database in `config\dev.exs` and run:
```

Comment:
You changed the other one, but at least it's consistent now 😄

Reply:
Hahah sorry, It was too late to think xD Fixed 👍